doc/langref.html.in

```diff
@@ -8229,6 +8229,34 @@ test "main" {
       {#see_also|@cVaArg|@cVaCopy|@cVaEnd#}
       {#header_close#}
 
+      {#header_open|@depositBits#}
+      <pre>{#syntax#}@depositBits(source: T, mask: T) T{#endsyntax#}</pre>
```

Comment:
Possibly? It would certainly simplify things in Sema. But should we really do that when we can absolutely do it at runtime, and that C and C++ support it iirc?